Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -96,9 +99,10 @@ QColor MappingIndicator::GetAltTextColor() const
   return palette().highlightedText().color();
 }
 
-QColor MappingIndicator::GetGateColor() const
+void MappingIndicator::AdjustGateColor(QColor* color)
```

Comment:
My only comment (and it's really getting nitpicky) is this seems like a better place for a reference.  Maybe just me but when I see a pointer, I expect `nullptr` to be a viable input.

Reply:
From https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md
"Functions that specifically modify their parameters should have the respective parameter(s) marked as a pointer so that the variables being modified are syntaxically obvious."

Reply:
Ah sorry, I was not aware of this.  Good to know.